lib/improver/tests/wind_downscaling/test_frictionvelocity.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
```

Comment:
The name of this file should be test_FrictionVelocity.py, so that the case of the letters matches the name of the class.